### ISSUE TITLE
Simple Translate unable to configure the target language

### ISSUE THREAD

**Issue by 3oheicrw** [snaildos/Fifo-Browser#100]
Even though the extension is installed and confirmed working, but I'm unable to configure it. Clicking on the extension's icon and choose to open setting only opened a blank error tab: fifo-error://network-error/-300

The language I'm speaking is not English so without being able to change the target language, the extension is almost useless for me.

**Comment by snaildos** [comment]
Issue with the extension not working with FEL, This is a hard fix, but I can change the language though, 
![image](https://user-images.githubusercontent.com/46210117/177079179-da356c9a-4254-4c48-a369-a5327a6c400e.png)
Anyway, will be working on FEL.
Closing issue, #104

**Comment by 3oheicrw** [comment]
> Issue with the extension not working with FEL, This is a hard fix, but I can change the language though, ![image](https://user-images.githubusercontent.com/46210117/177079179-da356c9a-4254-4c48-a369-a5327a6c400e.png) Anyway, will be working on FEL. Closing issue, #104

No. Doing this only changes the target language to the text you entered into this particular text field. If you select a block of texts on a random website and click to the popup to translate it, the target language still be English (this is the supposed usage of Simple Translate). If you right click on a random website and click Translate this page then Simple Translate still translate it into English. To really change the target language, you have to click to the gear icon next to the heart icon, which will open a new error tab on Fifo.

**Comment by snaildos** [comment]
Alright, thanks for the further clarification. Will re-open.

**Comment by snaildos** [comment]
Most probably related to chrome manifest v3.

**Comment by snaildos** [comment]
#104

**Comment by uddlrl** [comment]
To be clear it's unable to open it setting page. It's an invalid URL on Fifo.

**Comment by uddlrl** [comment]
> Most probably related to chrome manifest v3.

Are you sure about that?

**Comment by snaildos** [comment]
> 

Yup.

**Comment by snaildos** [comment]
> > Most probably related to chrome manifest v3.
> 
> Are you sure about that?

Most likely. Most extensions work, this does not. It's strange.

**Comment by 3oheicrw** [comment]
Dark Reader is one of the most annoying extension that doesn't really has an alternative. On Firefox, if you go to about:config and overwrite the user agent string with something from a browser not Firefox (in my case IE since I have to work with some sites require IE and I don't want to install addons just to do that). It turns out after that I'm unable to config Dark Reader (it's the same Loading. Please wait... when clicking on the addon's icon like on Fifo). After that I reset the user agent in about:config and install an user agent changer addon. Now I'm not very sure that it's manifest v3 or not. Dark Reader is just weird.

**Comment by pwd96m4a** [comment]
As whasmyuseragent.org shows Fifo is using the same user agent as Chrome so I don't think user agent has anything to do here.

**Comment by pwd96m4a** [comment]
> > > Most probably related to chrome manifest v3.
> > 
> > 
> > Are you sure about that?
> 
> Most likely. Most extensions work, this does not. It's strange.

It's not strange. Extensions installed and work (partially) but they all have problems with saving data, right? Dark Reader is more special as it works per site. It has per site settings to overwrite the global settings. Because it can't write and retrieve it data so it keeps loading. This is a bug of FEL, not something to blame Dark Reader for. I have no comment about the user agent changing broken Dark Reader on Firefox reported above though.

**Comment by snaildos** [comment]
Our "Extension Update" seems to be a bit of a "Extension Mess Up",
Even though FElv1 had more features and could allow extensions to save, FElv2 just broke most of the extensions.
Thinking about reverting FEl.

**Comment by 3oheicrw** [comment]
> Our "Extension Update" seems to be a bit of a "Extension Mess Up", Even though FElv1 had more features and could allow extensions to save, FElv2 just broke most of the extensions. Thinking about reverting FEl.

Since both of them are no longer developed now, reverting to FELv1 and fork it makes sense.
